src/bokeh/events.py

```diff
@@ -275,6 +277,22 @@ def __init__(self, model: Model | None) -> None:
     def event_values(self) -> dict[str, Any]:
         return dict(**super().event_values(), model=self.model)
 
+class AxisClick(ModelEvent):
+    ''' Announce a location where an axis was clicked.
+
+    '''
+    event_name = 'axis_click'
+
+    value: float | FactorType | None
+
+    def __init__(self, model: Axis | None, value: float | FactorType | None = None) -> None:
+        from .models import Axis
+        if model is not None and not isinstance(model, Axis):
+            clsname = self.__class__.__name__
+            raise ValueError(f"{clsname} event only applies to axis models")
+        super().__init__(model=model)
+        self.value = value
```

Comment:
Should this event return anything else? Would a screen coordinates or location along the axis be useful?

Reply:
Possibly. In fact this event could inherit from some kind of generic tap event to introduce a standard set of attributes for this class of events.

Reply:
I'm pretty ambivalent. Without a clear motivation I will punt on this for now.